CPAN/buildme.sh

```diff
@@ -799,6 +799,11 @@ function build {
                     patch -p0 < ../EV-fixes.patch # patch to disable pthreads and one call to SvREADONLY
                 fi
             fi
+	    if [ "$OS" = "SunOS" ]; then
+	    	cd libev
+		patch -p0 < ../../EV_fstyp.patch
+		cd ..
+	    fi
```

Comment:
Please make sure you use the same style indentation as the rest (spaces vs. tabs).

Reply:
whitespace cleanup done